client/StatusTable/components/StatusTable.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import {connect} from 'react-redux'
+import * as actions from '../actions'
+import WeekRow from './WeekRow'
+
+class StatusTable extends Component {
+    componentDidMount() {
+        this.props.tableLoadInit();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.startSelected !== prevProps.startSelected) {
+            this.props.tableLoadInit();
+        }
+    }
+
+    // before this we lose _id from parent component
+    // componentWillUnmount() {
+    //     const { _id } = this.props;
+    //     this.props.tableCloseInit({_id});
+    // }
+
+    render() {
+        const { workweeks }= this.props;
```

Comment:
missing  space after curly bracket. You can try linters to check and prettier to fix such things